packages/components/resource-pages/bucket-policy-list-view.tsx

```diff
@@ -211,10 +216,10 @@ export const BucketPolicyListView: React.FC = () => {
       resourceModel={NooBaaBucketClassModel}
       resourceMap={objectBucketClaimMap}
       tableColumns={tableColumns as TableColumn<K8sResourceCommon>[]}
-      createButtonTitle={t('Create bucket')}
+      createButtonTitle={t('Create new')}
```

Comment:
```suggestion
      createButtonTitle={t('Create new bucket policy')}
```
similarly for data source "Create new data source"